custom-namespaces.yaml

```diff
@@ -604,3 +604,21 @@ regrets_reporter:
       type: table_view
       tables:
         - table: moz-fx-data-shared-prod.regrets_reporter_derived.regrets_reporter_summary_v1
+user_journey:
+  glean_app: false
+  pretty_name: User Journey
+  owners:
+    - mviar@mozilla.com
+    - brennie@mozilla.com
+    - aoprea@mozilla.com
+    - dmosedale@mozilla.com
+    - elee@mozilla.com
+    - emcminn@mozilla.com
+    - pdahiya@mozilla.com
```

Comment:
@lmegviar These are all the users that will have write access to the folder. I looked them up via the People Directory, but feel free to add/remove people as necessary.

Reply:
@scholtzan I'd also add our engineering manager at tspurway@mozilla.com.